code/modules/modular_computers/file_system/programs/camera.dm

```diff
@@ -6,15 +6,15 @@
 	switch(network)
 		if(NETWORK_THUNDER)
 			return 0
-		if(NETWORK_ENGINE || NETWORK_ENGINEERING || NETWORK_ENGINEERING_OUTPOST || NETWORK_ALARM_ATMOS || NETWORK_ALARM_FIRE || NETWORK_ALARM_POWER)
+		if(NETWORK_ENGINE,NETWORK_ENGINEERING,NETWORK_ENGINEERING_OUTPOST,NETWORK_ALARM_ATMOS,NETWORK_ALARM_FIRE,NETWORK_ALARM_POWER)
```

Comment:
Not sure `if()` statements work that way?

Reply:
Yeah, that one is odd, but it was deliberately done in https://github.com/Baystation12/Baystation12/pull/14227

I am guessing something weird was going on there.

Reply:
Ooooh, it's switch syntax. How you specify multiple discrete cases, per byond documentation:
![docs](https://puu.sh/Ddxm0/4dff5223c3.png)